test/arrayops.jl

```diff
@@ -1734,6 +1734,10 @@ end
     b[CartesianIndex{2}(1,1)] = 7
     @test a[1,2] == 7
     @test 2*CartesianIndex{3}(1,2,3) == CartesianIndex{3}(2,4,6)
+    @test CartesianIndex{3}(1,2,3)*2 == CartesianIndex{3}(2,4,6)
+    @test_throws ErrorException iterate(CartesianIndex{3}(1,2,3))
+    @test CartesianIndices(CartesianIndex{3}(1,2,3)) == CartesianIndices((1, 2, 3))
+    @test Tuple{}(CartesianIndices{0,Tuple{}}(())) == ()
```

Comment:
Simpler as `Tuple(CartesianIndex()) == ()`.

Also worth noting that these days one can just say `CartesianIndex(1,2,3)`. Looks like a few of the tests haven't been updated to take advantage of what I'm presuming (I don't remember) are newer convenience methods.